webapp/packages/plugin-object-viewer/src/ObjectPropertiesPage/ObjectPropertyTable/MENU_OBJECT_VIEWER_FOOTER_ACTIONS.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createMenu } from '@cloudbeaver/core-view';
+
+export const MENU_OBJECT_VIEWER_FOOTER_ACTIONS = createMenu('object-viewer-footer-actions', 'Object viewer footer actions menu');
```

Comment:
please don't use `MENU` and `ACTIONS` in the same name because we use this key words for splitting menus and actions globally